src/auto-evo/RunResults.cs

```diff
@@ -309,6 +320,14 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
                     }
                 }
 
+                if (GetGlobalPopulation(entry.Species) == 0 &&
```

Comment:
I would prefer to have the check be like this: `GetGlobalPopulation(entry.Species) + popDuringMove <= 0`

Though it isn't fundamentally currently much different but sounds better to me.

Reply:
Easier to read

Reply:
I'm just saying that there could be a bug or something where global population is negative so the strict check you have now is not programmed in a defensive way to guard against other parts of the code being wrong.

Reply:
That too